src/components/CreateAnalysis/CreateAnalysisWrapper.tsx

```diff
@@ -7,19 +7,47 @@ import { DcmImage } from "../../context/reducers/dicomImagesReducer";
 import CreateAnalysisService from "../../services/CreateAnalysisService";
 import ConfirmAnalysis from './ConfirmAnalysis';
 import CreateAnalysisDetail from "./CreateAnalysisDetail";
+import pacs_integration from '../../services/pacs_integration';
+import chris_integration from '../../services/chris_integration';
 
 const CreateAnalysisWrapper = () => {
   const { state: { dcmImages, createAnalysis: { selectedStudyUIDs } }, dispatch } = useContext(AppContext)
   const [isExpanded, setIsExpanded] = useState(false);
   const [isModalOpen, setIsModalOpen] = useState(false);
   const history = useHistory();
 
-  const submitAnalysis = () => {
-    const imagesSelected: DcmImage[] = CreateAnalysisService.pickImages(dcmImages, selectedStudyUIDs);
+  const submitAnalysis = async () => {
+    let imagesSelected: DcmImage[] = CreateAnalysisService.pickImages(dcmImages.filteredDcmImages, selectedStudyUIDs);
```

Comment:
As a user, shouldn't I be able to select any image for analysis even if it doesn't appear after filtering?

Use case:
1) A user selects DICOM A for analysis
2) A user is looking for an older DICOM so they apply a filter to find the old one
3) A user finds that DICOM and selects it
4) A user then clicks 'Analyze' while the filter hasn't been reset
Wouldn't the user be able to analyze both images in that case?
```suggestion
    let imagesSelected: DcmImage[] = CreateAnalysisService.pickImages(dcmImages.allDcmImages, selectedStudyUIDs);
```

Reply:
Good point! Being able to submit files for analysis that come from different filter results is a more appropriate UX.